lib/crypto/doc/src/crypto.xml

```diff
@@ -835,6 +835,8 @@ end
 	<p>For decryption, set the <c>EncryptFlag</c> to <c>false</c> and put the tag to be checked
 	in the argument	<c>TagOrTagLength</c>.
 	</p>
+  <p>The <c>AAD</c> argument represents the Additional Authentication Data. It's optional. If it's
+  not needed, it can be set to an empty binary (<c>&lt;&lt;&gt;&gt;</c>).
```

Comment:
```suggestion
  not needed, it can be set to an empty binary (<c>&lt;&lt;&gt;&gt;</c>).</p>
```

Reply:
Oooph, how did I screw that up. Thank you @garazdawi! 💟